### DISCUSSION TITLE
Monitor Ultrawide? Qual monitor vocês usam?

### DISCUSSION THREAD

**Discussion by zecafeitosa** [frontendbr/forum#1616] [Perguntas e Respostas]
Estou querendo organizar meu "escritório" em casa e estou na dúvida de qual monitor comprar. Eu tenho um Macbook air de 13 polegadas.

Estava pensando em pegar um Monitor Ultrawide pra ficar usando apenas um. Indicaram o **Monitor LG LED 29´ Ultrawide**

O que vocês acham?

**Comment by eliseumds** [comment]
Eu não pegaria um de 29" pq não acho suficiente e, sendo curvo, não rola agrupar com outro monitor, por isso 34"+ atinge aquele sweet spot sendo quase equivalente a dois monitores normais. Se estiver com os bolsos pesados, eu fiquei surpreso com a qualidade do Samsung QLED de 49": https://www.youtube.com/watch?v=0jSxvjT6Ec0

**Comment by caioincau** [comment]
Eu ia somar exatamente isso pegaria um 34+

**Comment by evertonthepaula** [comment]
Ultrawide de 29 com notebook de 15 funciona super bem, sistemas abertos em 1920px/1960px, ainda sobra espaço para abrir o console na tela e código/terminal aberto no Notebook.

Já no seu caso acho a tela de 13 muito pequena, usaria como cotado acima uma tela maior.

**Comment by MathCunhaDev** [comment]
Eae,

Cara peguei um LG 29wk600 esse mês e não me arrependo, pensei que ia ser pequeno mas agora acho que maior poderia me dar problemas para visualizar tudo se fosse maior.

**Comment by eliseumds** [comment]
@MatCunha pois é, tem gente que não precisa de tanto espaço. @zecafeitosa se você está acostumado com menos espaço, um upgrade para um 29" + a tela do laptop não vai ser ruim não.

**Comment by leo-tavares** [comment]
Eu uso o lg25um58, é modelo ultrawide de 25". Programo e jogo com ele a mais ou menos um ano, no começo eu acho ele muito baixo, a altura dele parece um monitor de 18".Em compensação e largura de tela realmente fez diferença principalmente para splitar o vscode em 2/3 arquivos de uma vez. No geral valeu a aquisição foi muito benéfica. 
No teu caso a experiência será ainda melhor até pq vc pretende comrpar um de 29".

**Comment by kivervinicius** [comment]
Aproveitando, alguém usa TV como monitor? Apesar dos hz serem baixos no momento precisaria de algo assim para conseguir usar das duas formas, se sim, alguma indicação?

**Reply by guilhermealveslopes** [reply]
> Aproveitando, alguém usa TV como monitor? Apesar dos hz serem baixos no momento precisaria de algo assim para conseguir usar das duas formas, se sim, alguma indicação?

Já viu aqueles projetores do Aliexpress? eles podem ser uma boa alternativa a TV em si.
Ai você separa do seu setup e sem gastar muito.

**Comment by imarinho-dev** [comment]
Pq não pega um monitor 4K? Não adianta ter um monitor enorme se a resolução é a mesma.

**Comment by weltonsanches** [comment]
Eu também estou considerando pegar esse monitor, mas estou em um grande dilema entre o 29" ultrawide ou 2 monitores de 23".

Pelo que vi a altura do 29" ultrawide e o 23 são praticamente as mesmas (o 23 é pouca coisa mais alto) e preço também não da muita diferença. O que acham?

**Comment by tbooom** [comment]
eu tenho o monitor lg uwide e estou satisfeito, uso pra programar e é muito bom poder dividir a tela em 2 janelas sem ficar apertado, as cores são muito bonitas e fiéis ( bom pra quem é designer ).
Segue print de como fica organizado ( https://prnt.sc/qk1q5l ).
Edit: o print é de 2 monitores, o da esquerda é só pra assistir as video-aulas.

**Comment by felipe964** [comment]
Olá!
Gostaria de atualizar o tópico)
Qual monitor é melhor agora? talvez um [ desses ](https://pt.globy.com/b2bmarket/listing/consumer-electronics/laptops-computers-accessories) monitores?

**Comment by eliseumds** [comment]
@felipe964 hoje existe muita opção de monitor 4K decente. Se vc trabalha com imagem/vídeo e precisa de alta fidelidade nas cores, aí a conversa muda. Eu pessoalmente uso um Asus ProArt PA279CV 27" que comprei no ano passado por ~3700 reais, mas um LG assim de 1863 reais é suficiente pra 99% das pessoas: https://www.kabum.com.br/produto/472913/monitor-profissional-lg-27-4k-uhd-ips-displayport-e-hdmi-hdr400-freesync-dci-p3-95-branco-27up650-w

**Reply by felipe964** [reply]
Obrigado!
eu vou assistir
